src/common/ipc.ts

```diff
@@ -127,6 +68,9 @@ export function broadcastIpc({ channel, webContentId, filter, args = [] }: IpcBr
   views.forEach(webContent => {
     const type = webContent.getType();
     logger.debug(`[IPC]: broadcasting "${channel}" to ${type}=${webContent.id}`, { args });
-    webContent.send(channel, ...[args].flat());
+    webContent.send(channel, ...args);
+    if (frameId) {
+      webContent.sendToFrame(frameId, channel, ...args)
+    }
```

Comment:
So if there is a `frameId` we send it twice?

Reply:
nope, first to BrowserWindow (main window)